### DISCUSSION TITLE
This is urgent

### DISCUSSION THREAD

**Discussion by undyingkevin** [serenity-is/Serenity#7279] [Help]
I have StartSharp.Template_v8.7.1.0.vsix. When I create startsharp template using this extension in Visual Studio, I run into package issues. 
How can I resolve this problem?
![error](https://github.com/user-attachments/assets/bb74c5f9-4b1b-429e-8983-c33938aff438)

**Comment by VictorTomaili** [comment]
follow the instructions in your membership dashboard for nuget package repository access. And ask startsharp related questions in startsharp repository or in email.

**Comment by undyingkevin** [comment]
Could you nicely tell me in more detail?

**Reply by VictorTomaili** [reply]
If you’re a StartSharp member, please log in to the members' area and check the dashboard for more details. If you're not a member, access to these packages is restricted.
